src/operations/graph/graphHelpers.js

```diff
@@ -993,5 +993,5 @@ async function processGraph(
 }
 
 module.exports = {
-    processGraph: processGraph
+    processGraph
```

Comment:
I'm not certain this is an issue, but I've always seen the processGraph: processGraph notation in these exports. Is the second piece being omitted problematic?

Reply:
no, it is just unnecessary.

Reply:
as long as the public name matches the private name it does not need to be specified twice.  I did not know that until recently :-)